### PR TITLE
12285-FSS UI blank field defaults to @Businessunit in query.

### PR DESCRIPTION
**Pull request by Shrikan14443**
FSS UI Client-side validation when the user keeps a blank field and highlights the blank field.

### CODE REVIEW

File: src/app/core/services/fss-search-validator.service.ts [hunk 1]
```diff
@@ -39,12 +40,22 @@ export class FssSearchValidatorService {
           }
         }
       }
+      if (!fieldDataType) {
+        if (fssSearchRows[rowId].selectedField === "" || fssSearchRows[rowId].selectedOperator === "") {
+          if (fssSearchRows[rowId].fieldFormControl.touched === false) {
+            fssSearchRows[rowId].fieldFormControl.markAsTouched();
+          }
+        }
+        this.errorMessageTitle = "There is a problem with a Field";
```
Comment (crdandoukho) [inline src/app/core/services/fss-search-validator.service.ts:49]: Please change title to 
`There is a problem with a field`

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Just a simple text change please

**Review by crdandoukho — APPROVED** [review]
Thanks for the change. Approved

File: src/app/core/services/fss-search-validator.service.ts (hunk 1 above)
Comment (Shrikan14443) [reply src/app/core/services/fss-search-validator.service.ts]: Hi Chris,
I have made the requested change.

**Review by Rashi14748 — APPROVED** [review]
looks good to me

**Review by martyn-fewtrell — APPROVED** [review]
Thanks for the updates

**Pull request merged**